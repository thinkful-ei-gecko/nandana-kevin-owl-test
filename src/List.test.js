import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './List';

describe('<List />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List participants={[]} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('list renders as expected', () => {
    const tree = renderer.create(<List participants={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});