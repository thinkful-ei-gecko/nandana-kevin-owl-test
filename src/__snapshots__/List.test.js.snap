// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> list renders as expected 1`] = `
<div
  className="List"
/>
`;
